packages/tools/src/store/removeEnabledElement.ts

```diff
@@ -46,6 +47,7 @@ function removeEnabledElement(
   // Listeners
   mouseEventListeners.disable(element);
   wheelEventListener.disable(element);
+  touchEventListeners.disable(element);
```

Comment:
why aren't we adding the disable for the touchEventDispatcher below?

Reply:
good catch will add